docs/src/props-table.js

```diff
@@ -78,7 +78,7 @@ function BasePropRows({passthroughPropsLink, elementType, isPolymorphic, refType
     <>
       <SxRow />
       {isPolymorphic && <AsRow defaultElementType={elementType} />}
-      <RefRow refType={refType} isPolymorphic={isPolymorphic} />
+      {refType && <RefRow refType={refType} isPolymorphic={isPolymorphic} />}
```

Comment:
guarding this so that it doesn't render this row unless a passed as a param. I think this is useful when the underlying component doesn't support refForwarding. cc @colebemis

Reply:
👍